### PR TITLE
Dead end S.Security.Cryptography.OpenSsl package

### PR DESCRIPTION
**Pull request by ViktorHofer**
To reduce complexity of the cryptography OpenSsl library, dead ending
its package as most of the configurations are partial facades anyway and
adding OpenSsl to the targeting pack. It's already part of the runtime
and exposed in aspnetcore's targeting pack. Note that the Platform
Analyzer will discourage the usage of these APIs on unsupported platforms.

.NET Standard libraries can continue to use the latest available package
which harvests for .NET Standard configurations.

Contributes to https://github.com/dotnet/runtime/issues/47530

### CODE REVIEW

File: src/libraries/pkg/baseline/packageIndex.json [hunk 1]
```diff
@@ -6153,32 +6153,6 @@
         "4.0.1.0": "4.3.0"
       }
     },
-    "System.Security.Cryptography.OpenSsl": {
```
Comment (ericstj) [inline src/libraries/pkg/baseline/packageIndex.json:6156]: Why removing entirely? We need to make sure this remains and any existing usages for downlevel frameworks use the past stable packages.

Comment (ViktorHofer) [reply src/libraries/pkg/baseline/packageIndex.json:6156]: We don't have any references to OpenSsl in non live built frameworks today. I wasn't sure if we want to leave or remove it. Will add it back.

Comment (ericstj) [reply src/libraries/pkg/baseline/packageIndex.json]: Please also add
```json
"InboxOn": {
  "net6.0": "6.0.0.0"
}
```

Comment (ViktorHofer) [reply src/libraries/pkg/baseline/packageIndex.json]: done, thanks

**Review by ericstj — APPROVED** [review]
LGTM modulo the cleanup that @bartonjs might want to do.

**Review by bartonjs — APPROVED** [review]

**Pull request merged**